package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "20-intro_to_angular_and_webpack",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --progress --colors",
+    "watch": "webpack-dev-server --progress --colors --inline --hot",
+    "lint": "eslint app/*.js",
+    "test-watch": "./node_modules/karma/bin/karma start",
+    "test": "./node_modules/karma/bin/karma start --single-run"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jonathanheemstra/20-intro_to_angular_and_webpack.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/jonathanheemstra/20-intro_to_angular_and_webpack/issues"
+  },
+  "homepage": "https://github.com/jonathanheemstra/20-intro_to_angular_and_webpack#readme",
+  "devDependencies": {
+    "angular-mocks": "^1.6.1",
+    "eslint": "^3.13.1",
+    "jasmine-core": "^2.5.2",
+    "karma": "^1.4.0",
+    "karma-jasmine": "^1.1.0",
+    "karma-mocha-reporter": "^2.2.1",
```

Comment:
Way to add this to make the tests more readable when you run them in the terminal!